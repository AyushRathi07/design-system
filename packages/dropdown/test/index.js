import React from 'react'
import renderer from 'react-test-renderer'
import {
  Dropdown,
  DropdownButton,
  DropdownList,
  DropdownItem,
  DropdownLink
} from '../src'

describe('Dropdown', () => {
  test('renders', () => {
    const json = renderer
      .create(
        <Dropdown identifier="test-dropdown">
          <DropdownButton>Uncontrolled Dropdown</DropdownButton>
          <DropdownList>
            <DropdownLink>Test Link</DropdownLink>
            <DropdownItem>One</DropdownItem>
            <DropdownItem>Two</DropdownItem>
            <DropdownItem>Three</DropdownItem>
          </DropdownList>
        </Dropdown>
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
