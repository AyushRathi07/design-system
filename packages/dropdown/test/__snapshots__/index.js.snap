// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders 1`] = `
.c0 {
  font-size: 12px;
  font-weight: 500;
  background-color: transparent;
  padding: 12px;
  text-align: left;
}

.c0:hover {
  cursor: pointer;
}

<button
  aria-expanded={false}
  aria-haspopup="menu"
  className="c0"
  data-reach-menu-button={true}
  id="test-dropdown-button"
  onClick={[Function]}
  onKeyDown={[Function]}
  onMouseDown={[Function]}
  type="button"
>
  Uncontrolled Dropdown
</button>
`;
